spec/consensus/light-client.md

```diff
@@ -8,7 +8,7 @@ In order to make sure that full nodes have the incentive to follow the protocol,
 
 1. The lite client needs a method to verify headers it obtains from full nodes according to trust assumptions -- this document.
 
-2. The lite client must be able to connect to one correct full node to detect and report on failures in the trust assumptions (i.e., conflicting headers) -- a future document.
+2. The lite client must be able to connect to one correct full node to detect and report on failures in the trust assumptions (i.e., conflicting headers) -- this document.
 
```

Comment:
I guess it would make sense to split the spec into two documents: (1) verification/bisection and (2) detector, as the functionality is quite different. (cp. https://github.com/interchainio/tendermint-rs/blob/e2cb9aca0b95430fca2eac154edddc9588038982/docs/architecture/adr-002-lite-client.md)